src/commands/edge_app_server.rs

```diff
@@ -259,4 +322,21 @@ settings:
 
         assert_eq!(resp.status(), 404);
     }
+
+    #[tokio::test]
+    async fn test_mock_data_from_str() {
```

Comment:
Please improve the naming - see here how we name our tests https://phorge.wireload.net/w/developer_guidelines/tests/

should contain what it tests and the expected behavior (but see the document anyway for in depth explanation and examples)

Reply:
Done!